class PyGlobe3DError(Exception):
    pass
